src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java

```diff
@@ -591,10 +591,9 @@ public AboutToStartOrSubmitCallbackResponse aboutToSubmitCaseCreation(
             }
             if (caseDataUpdated.get(CASE_TYPE_OF_APPLICATION) != null) {
                 caseDataUpdated.put("selectedCaseTypeID", caseDataUpdated.get(CASE_TYPE_OF_APPLICATION));
-                if (launchDarklyClient.isFeatureEnabled("task-list-v2")) {
-                    if (C100_CASE_TYPE.equals(caseDataUpdated.get(CASE_TYPE_OF_APPLICATION))) {
-                        caseDataUpdated.put("taskListVersion", TASK_LIST_VERSION_V2);
-                    }
+                if (launchDarklyClient.isFeatureEnabled("task-list-v2")
```

Comment:
do we need this feature enabled condition now? as this is already in prod.

Reply:
This is to support backdated draft application. Ideally this should be removed by PET team and its not in scope of this ticket.

Reply:
yes.. agreed with you that this should be taken up by PET team. However, the ld check will still return true for backdated drafted case.